src/Entities/Minecart.cpp

```diff
@@ -698,6 +728,20 @@ void cMinecart::SnapToRail(NIBBLETYPE a_RailMeta)
 
 
 
+bool cMinecart::TestBlockCollision(int a_XOffset, int a_YOffset, int a_ZOffset)
```

Comment:
For the performance? Usually compilers already do a good job deciding if a function should be inlined.